src/Features/Core/Portable/Completion/CompletionServiceWithProviders.cs

```diff
@@ -96,7 +96,12 @@ private ImmutableArray<CompletionProvider> GetAllProviders(ImmutableHashSet<stri
                 .Where(lz => lz.Metadata.Roles == null || lz.Metadata.Roles.Length == 0 || roles.Overlaps(lz.Metadata.Roles))
                 .Select(lz => lz.Value);
 
-            return imported.ToImmutableArray();
+#pragma warning disable 0618
+            // We need to keep supporting built-in providers for a while longer since this is a public API.
+            var builtin = GetBuiltInProviders();
```

Comment:
We should file an issue tracking that TS/javascript and F# are broken by this removal and track them moving to MEF exports.

Reply:
https://github.com/dotnet/roslyn/issues/42367

Reply:
TS/F# should not be broken by this change. The MEF approach has been around for the same time that `GetBuiltInProviders` has been around. There is no need to support the latter since the partners can fix their exports even without waiting on a Roslyn change.